web/src/scss/content/map/_map.scss

```diff
@@ -1,12 +1,3 @@
-#main {
```

Comment:
I'm super worried about removing this (as I remember testing on many devices etc..) - what is the reason? Do we have equivalent code copied somewhere? Maybe it's worth double checking in the scss PR to understand what happened.

Reply:
Ok so I think this is where it all started: https://github.com/tmrowco/electricitymap-contrib/pull/2111/files#diff-a344ab15f45451bd3fe93462dc8325d1L111-L112 (index.ejs)
![image](https://user-images.githubusercontent.com/1216874/77055032-9f99eb00-69d0-11ea-8950-d3cc964c9310.png)

So the container ID was renamed `main` -> `app` which I assume broke `OnboardingModal` at the time and also made this SCSS block unused.

I actually tried applying this SCSS to `#app`, but things rendered weirdly for me so I thought it'd be ok to remove it as it currently seems unused.

Of course, there is a chance I'm wrong and that `#main` is somehow set implicitly somewhere (even though I couldn't find it anywhere else in the code), so if you want to be on a very safe side, I can revert this bit and just keep the change to the `OnboardingModal` mounting point - WDYT?

Reply:
Let's keep it removed, as right now, on the production, document.getElementById('main') returns null -- so this style is unused.